tests/test_store/test_stateful.py

```diff
@@ -28,11 +27,7 @@ def mk_test_instance_sync() -> None:
 
     if isinstance(sync_store, ZipStore):
         pytest.skip(reason="ZipStore does not support delete")
-    elif isinstance(sync_store, LocalStore):
-        pytest.skip(reason="This test has errors")
-    elif isinstance(sync_store, MemoryStore):
-        run_state_machine_as_test(mk_test_instance_sync, settings=Settings(max_examples=50))
-    else:
-        run_state_machine_as_test(
-            mk_test_instance_sync, settings=Settings(report_multiple_bugs=True)
-        )
+
+    if isinstance(sync_store, LocalStore):
+        pytest.skip(reason="Test isn't suitable for LocalStore.")
```

Comment:
Could you expand the reason here a bit to explain why it's not suitable? Just thinking of down the line when someone tries to understand this a bit more info would be helpful.